### PR TITLE
Issue #22 profile page

### LINKED ISSUE #22: Profile Page
**Overview**

Specs as specified from [realworld](https://github.com/gothinkster/realworld):

> * Profile page (URL: /#/profile/:username, /#/profile/:username/favorites )
        * Show basic user info
        * List of articles populated from author's created articles or author's favorited articles

**Requirements**

* Develop the views (including Vue), JS (if needed) and CSS (if needed) for `/profile/:username` routes
    * Utilise the frontend styling as specified [here](https://github.com/gothinkster/realworld-starter-kit/blob/master/FRONTEND_INSTRUCTIONS.md)
    * File structure is unknown currently, for example a single Vue file could handle the /profile/:username and /profile/:username/favorites routes
* Develop the resource required to handle this
_Must meet the backend API specs [here](https://github.com/gothinkster/realworld/tree/master/api)_
* Create a new database table named `profiles` so that it returns the following model:
    ```
    {
      "username": "jake",
      "bio": "I work at statefarm",
      "image": "https://static.productionready.io/images/smiley-cyrus.jpg",
      "following": false
    }
    ```
    See https://github.com/gothinkster/realworld/tree/master/api#profile

**Optional**
* Develop tests (can be seperated into another issue)

**Related Files**
* `/.docker/data/postgres_dump.sql`
* `/src/resources/profiles_resource.ts`
* `/src/vue/routes/Profile.vue`

### PR DESCRIPTION
**Pull request by crookse**
Closes #22

### CODE REVIEW

**Comment by crookse** [comment]
Weird. I will try and get back to this. Should be an easy fix.

**Comment by crookse** [comment]
Check the latest commit, @Guergeiro. The form fields are reset before the page is loaded now.

File: src/vue/routes/Login.vue [hunk 1]
```diff
@@ -84,5 +84,12 @@ export default {
         });
     }
   },
+  beforeRouteEnter(to, from, next) {
+    next((vm) => {
+      console.log("Resetting login form fields.");
+      vm.email = null;
+      vm.password = null;
+    });
+  }
```
Comment (Guergeiro) [inline src/vue/routes/Login.vue:93]: Also implement this for signup form.

Comment (crookse) [reply src/vue/routes/Login.vue:93]: I willl make an issue for that

**Pull request merged**